smithy-model/src/test/resources/software/amazon/smithy/model/errorfiles/loader/dupe-list-member-names.errors

```diff
@@ -1 +1 @@
-[ERROR] com.foo#List: Parse error at line 7, column 11 near `: `: Duplicate member of com.foo#List: 'member' | Model
+[ERROR] -: Parse error at line 7, column 5 near `member`: Expected: '}', but found 'm' | Model
```

Comment:
Will this be added back later? This seems like a pretty important error to display.

There seems to be a lot of tests files with similar changes.

Reply:
This is the same test case just caught in the parser rather than after parsing and through logic. Previoulsy we parsed a bunch of members into a map and then validation. Now we can just parse known members one at a time.